### PR TITLE
feat: add `astronvim` editor client

### PR DESCRIPTION
**Pull request by mehalter**
This adds the AstroNvim distribution to the available clients (we already maintain our own discord client ID)

### CODE REVIEW

File: assets/editor/astronvim.png
Comment (vyfor) [inline assets/editor/astronvim.png:1]: enlarged the background size so the entire logo fits within the rich presence. could you replace it?

astro

File: src/lib.rs [hunk 1]
```diff
@@ -72,6 +72,10 @@ pub extern "C" fn init(
                 1220296082861326378,
                 format!("{}/editor/nvchad.png", GITHUB_ASSETS_URL),
             ),
+            "astronvim" => (
+                1009122352916857003,
```
Comment (vyfor) [inline src/lib.rs:76]:
```suggestion
                1230866983977746532,
```

Comment (vyfor) [reply src/lib.rs:76]: we do maintain a Discord Apps Team of our own for Cord, is it fine by you?

**Review by vyfor — CHANGES_REQUESTED** [review]
could you please apply the following changes?

File: src/lib.rs (hunk 1 above)
Comment (mehalter) [reply src/lib.rs]: Updated the commit!

File: assets/editor/astronvim.png
Comment (mehalter) [reply assets/editor/astronvim.png:1]: Updated in the revised commit! Thanks!

**Comment by vyfor** [comment]
thanks! glad to be a part of the community

**Pull request merged**